ci_framework/playbooks/02-infra.yml

```diff
@@ -22,6 +22,10 @@
     - name: Prepare container package builder
       ansible.builtin.include_role:
         name: pkg_build
+    - name: Hive actions
```

Comment:
Not sure if this is the right place

Reply:
Yes, this is the right place. In the end, we expect a cluster ready and a KUBECONFIG as output.

Reply:
@psathyan ack, I changed the structure in the next commit.